tests/unittests/core/types/pointtest.cpp

```diff
@@ -156,6 +156,86 @@ INSTANTIATE_TEST_SUITE_P(PointRotateTest, PointRotateTest, ::testing::Values(
 ));
 // clang-format on
 
+/*******************************************************************************
+ *  Tests for getLength()
+ ******************************************************************************/
+
+struct PointLengthPrecisionTestData {
+  Point input;
+  UnsignedLength length;
+};
+
+class PointLengthPrecisionTest : public ::testing::TestWithParam<PointLengthPrecisionTestData> {};
+
+TEST_P(PointLengthPrecisionTest, testPrecision) {
+  const PointLengthPrecisionTestData& data = GetParam();
+
+  Point p(data.input);
+  EXPECT_NO_THROW(p.getLength());
+  EXPECT_EQ(data.length, p.getLength()) << "Computed: " << p.getLength()->toNm() << " Expected: " << data.length->toNm();
```

Comment:
Yes I will reformat this issue, I need to know how long line You do accept ? 80 chars ? 

I don't like autoformat scripts because they can crush the code. It's better that Editor should do this job, and I see all issues and correct manually. There is bug in your attached linter inside repository so I can't use CLion linter.

```
Error reading ...LibrePCB/.clang-format:100:4: Invalid argument - unknown key 'Delimiter'
```

Reply:
> There is bug in your attached linter inside repository so I can't use CLion linter.

It's not a bug, it's because `clang-format` is not compatible between different versions. That's exactly the reason why our formatting script allows to run the correct version of `clang-format` in a Docker container. This is the only way to make sure all developers use the same coding style. So please please don't format in any other way, use the provided script (just configure a keyboard shortcut in your IDE to call the script, it's so easy). Or as I mentioned, I could add a formatting commit before merging if desired.